fxa-auth-db-server/index.js

```diff
@@ -89,6 +89,7 @@ function createServer(db) {
       return db.deleteEmail(req.params.id, req.params.email)
     })
   )
+  api.get('/account/emails/:id', withIdAndBody(db.getEmail))
```

Comment:
above we have `api.del('/account/:id/emails/:email',` should this here be `:email`, instead of `:id`?